perun-wui-core/src/main/java/cz/metacentrum/perun/wui/json/ErrorTranslator.java

```diff
@@ -49,7 +49,7 @@ public static String getTranslatedMessage(PerunException error) {
 			GeneralObject holder = error.getAttributeHolder();
 			GeneralObject secondHolder = error.getAttributeHolderSecondary();
 
-			String text = "Wrong value of attribute (value or format).<p>";
+			String text = a.getFriendlyName().startsWith("ssh") ? error.getMessage() : "Wrong value of attribute (value or format).<p>";
```

Comment:
I tested it locally and Exception doesn't contains Attribute object. Hence you can't call `a.getFriendlyName()` if `a` is `NULL` and you must check for it before doing so.